src/darwin/Framework/CHIP/CHIPCluster.h

```diff
@@ -33,6 +33,38 @@ NS_ASSUME_NONNULL_BEGIN
 + (instancetype)new NS_UNAVAILABLE;
 @end
 
+/**
+ * CHIPCommandArguments
+ *    This is used to control the behavior of cluster commands.
+ *    If not provided (i.e. nil passed for the CHIPCommandArguments argument), will be
+ *    treated as if no arguments were passed in.
+ */
+@interface CHIPCommandArguments : NSObject
+
+/**
+ *
+ */
+@property (strong, nonatomic, nullable) NSNumber * timedInvokeTimeoutMs;
+
+- (instancetype)init;
+@end
+
+/**
+ * CHIPWriteParams
+ *    This is used to control the behavior of cluster writes.
+ *    If not provided (i.e. nil passed for the CHIPWriteParams argument), will be
+ *    treated as as if no arguments were passed in.
+ */
+@interface CHIPWriteParams : NSObject
+
+/**
+ *
+ */
+@property (strong, nonatomic, nullable) NSNumber * timedWrite;
```

Comment:
Needs to be documented.  And is there a reason this is not a `timedWriteTimeoutMs`?

Reply:
Updated. Thank you.